### PR TITLE
Login Page Made

### PR DESCRIPTION
**Pull request by Asypher00**
check Login.css for code 
Screenshot (24)

### CODE REVIEW

File: Login.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.nav-footer{
+    align-items: center;
+    width : 20%;
+}
+.btn{
+    margin : 0px;
+    background-color: lightgray;
+}
+
+.nav-main{display: flex;
```
Comment (coldpigli) [inline Login.css]: indentation not correct

File: Login.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+body{background-color: white;
+    color : black;
```
Comment (nishasen) [inline Login.css]: use variables here, and start properties from a new line

**Review by coldpigli — APPROVED** [review]
Left a suggestion. Everything else looks good.

File: Login.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items :center;
+}
+.nav-container{
+    background-color: rgb(188, 251, 62);
+ align-items: center;
+justify-content: space-between;
+width : 97vw;
```
Comment (nishasen) [inline Login.css]: give indentation

File: Login.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    align-items: center;
+    margin-left : 250%;
+    
+}
+
+.nav-input{text-align: center;
+    font-family: "comic sans ms";
+    color : black;
+    font-size: 1.25rem;
+    width : 500px;
```
Comment (Gautam2305) [inline Login.css]: convert to rems here for responsiveness.....tip: never use px if the value is greater than 5 or 10px depending on use-case

File: Login.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    width : 20%;
+}
+.btn{
+    margin : 0px;
+    background-color: lightgray;
+}
+
+.nav-main{display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (nishasen) [inline Login.css]: use util class

File: Login.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    border-radius: 10px;
+   }
+   
+   .login-container{
+       width : 20vw;
+       align-items: center;
+       display: flex;
+       flex-direction: column;
+       background-color:rgb(188, 251, 62) ;
+       margin-top : 200px;
```
Comment (Gautam2305) [inline Login.css]: here too... use rems

File: Login.css [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items :center;
+}
+.nav-container{
+    background-color: rgb(188, 251, 62);
```
Comment (aishwarya-bm) [inline Login.css]: create & use variable for this color

**Review by Gautam2305 — COMMENTED** [review]
suggested some changes

File: Login.css [hunk 8]
```diff
@@ -0,0 +1,91 @@
+body{background-color: white;
+    color : black;
+    font-family: "comic sans ms";
+    
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items :center;
```
Comment (rittik-ghosh) [inline Login.css]: common utility class could have been used

**Review by aishwarya-bm — APPROVED** [review]
All good, left a comment for color variable

**Review by nishasen — APPROVED** [review]
Suggested some changes, please look into the comments

**Review by rittik-ghosh — COMMENTED** [review]
I have a few suggestions that can be implemented later:-
1) Use more utility classes
2) Indent files or use "beautify"
3) Use variables with good explanatory names

**Pull request merged**